languagetool-language-modules/en/src/main/java/org/languagetool/language/English.java

```diff
@@ -105,7 +107,12 @@ public String getShortCode() {
   @NotNull
   @Override
   public Tagger createDefaultTagger() {
-    return new EnglishTagger();
+    WeakReference<EnglishTagger> ref = cachedTagger;
+    EnglishTagger tagger = ref == null ? null : ref.get();
+    if (tagger == null) {
+      cachedTagger = new WeakReference<>(tagger = new EnglishTagger());
```

Comment:
Could these be split into two lines? I think it's a bit uncommon to have a second assignment like this.

Reply:
It's common for me, but I see your point

Reply:
Force-pushed